sktime/utils/plotting/__init__.py

```diff
@@ -75,6 +77,25 @@ def plot_series(*series, labels=None):
 
         plot_func(x=x, y=y, ax=ax, marker="o", label=label, color=color)
 
+    # plot prediction intervals if present
+    if pred_int is not None:
+        # check same conditions as for earlier indices
+        if not type(index) is type(pred_int.index):  # noqa
+            raise TypeError("Found series with different index types.")
+        elif all([x in index for x in pred_int.index]):
+            plt.fill_between(
+                pred_int.index,
+                pred_int.lower,
+                pred_int.upper,
+                alpha=0.3,
+                color=ax.get_lines()[-1].get_c(),
+                label="95% prediction intervals",
```

Comment:
Alpha may be different so we should assign a more general label here:
```suggestion
                label="prediction intervals"
```

Reply:
Makes sense!